packages/server/lib/project_static.ts

```diff
@@ -0,0 +1,200 @@
+import Debug from 'debug'
```

Comment:
These were all static functions on `ProjectBase` that did not need to be there. `Project` represents a single instance of a project - these methods to not, so I moved them to a separate module to separate the concerns.

Reply:
NOTE: The file *above* this is "Large diffs are not rendered by default", but it's the most important file in this PR, so you should expand it and take a look.


It simplifies `ProjectBase` by moving each step (eg, `initializePlugins`, `startWebsockets`, etc) into separate functions with clear cut boundaries and responsibilities.

It also reduces the amount of times `config` is passed around and mutated significantly.